Library/Homebrew/keg_relocate.rb

```diff
@@ -12,7 +12,7 @@ class Keg
   class Relocation
     extend T::Sig
 
-    RELOCATABLE_PATH_REGEX_PREFIX = /(?<![a-zA-Z0-9])/.freeze
+    RELOCATABLE_PATH_REGEX_PREFIX = /(?:(?<=-I|-L|-isystem)|(?<![a-zA-Z0-9]))/.freeze
```

Comment:
Ah, we also want `-F` for framework includes.

Reply:
```suggestion
    RELOCATABLE_PATH_REGEX_PREFIX = /(?:(?<=-F|-I|-L|-isystem)|(?<![a-zA-Z0-9]))/.freeze
```